specification/inference/delete/DeleteResponse.ts

```diff
@@ -17,8 +17,10 @@
  * under the License.
  */
 
-import { AcknowledgedResponseBase } from '@_types/Base'
+import { DeleteInferenceEndpointResult } from '@inference/_types/Results'
 
 export class Response {
-  body: AcknowledgedResponseBase
+  body: {
```

Comment:
Hmm I think this should be:

```
export class Response {
  body: DeleteInferenceEndpointResult
}
```

I don't believe there's a `dryRunResult` field in the response.